src/test/resources/org/rust/ide/inspections/fixtures/unnecessary_parenthesis.rs

```diff
@@ -0,0 +1,9 @@
+fn main() {
+    let x = 5;
+
+    if <warning descr="Unnecessary parenthesis">(x == 5)</warning> {
+        println!("x is five!");
+    } else {
+        println!("x is not five :(");
+    }
+}
```

Comment:
there is no need to remove the parenthesis around `true` and `false` in this if expression? @jajakobyly @matklad 

```rust
if (true) or (false) {}
```